recipes-core/rcu-sanity-check/files/post-update-sanity-check

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+rm /data/rcu-service/firmware/update-staged
+rm /etc/rcu-mender-image-firstboot
+
+RCU_SERVICE_STATUS=$(systemctl is-active rcu-service)
+ETH0_STATUS=$(cat /sys/class/net/eth0/operstate)
+
+if [ "$RCU_SERVICE_STATUS" = "active" ]; then
+    mender commit
+    touch /data/rcu-service/firmware/update-succeeded
+    systemctl restart rcu-service
+else
+    mender rollback
+    touch /data/rcu-service/firmware/update-failed
```

Comment:
A note to review whether we want the script to reboot RCU in this case.

Reply:
I'm in favor of rebooting RCU in case of update failure. Reason is because if the update fails, ethernet may not be enabled, and hence users won't know what happened to the rack, and will also be unable to trigger reboot remotely.